rialto_airflow/publish/publication.py

```diff
@@ -2,17 +2,52 @@
 from csv import DictWriter
 from pathlib import Path
 
-from sqlalchemy import select, func
-
-from rialto_airflow.database import get_session, Publication, Author, Funder
+from sqlalchemy import select, func, Boolean, Column, Integer, String
+from sqlalchemy.dialects.postgresql import insert
+from sqlalchemy.orm import declarative_base  # type: ignore
+from sqlalchemy.types import DateTime
+
+from rialto_airflow.database import (
+    RIALTO_REPORTS_DB_NAME,
+    create_schema,
+    get_session,
+    utcnow,
+    Publication,
+    Author,
+    Funder,
+)
 from rialto_airflow.utils import get_types, get_csv_path, normalize_pmid
 import rialto_airflow.publish.publication_utils as pub_utils
 
 
+ReportsSchemaBase = declarative_base()
+
+
+class PublicationAugmented(ReportsSchemaBase):  # type: ignore
+    __tablename__ = "publication_augmented"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    doi = Column(String, unique=True)
+    pub_year = Column(Integer)
+    apc = Column(Integer)
+    open_access = Column(String)
+    types = Column(String)
+    funders = Column(String)
+    federally_funded = Column(Boolean)
+    academic_council_authored = Column(Boolean)
+    faculty_authored = Column(Boolean)
+    created_at = Column(DateTime, server_default=utcnow())
+    updated_at = Column(DateTime, onupdate=utcnow())
```

Comment:
I'm not sure we need `created_at` and `updated_at` for reporting, but it seems harmless (and maybe useful) to have them.